docs/architecture/decision-records/GithubActionWorkflow.md

```diff
@@ -0,0 +1,23 @@
+# GitHub Action (GHA) Reusable Workflows
+
+## Context
+
+The main GHA workflow file (continuous-deployment.yml) had grown in size and complexity. To improve readability and maintainability job steps will be grouped into reusable workflows. To use a GHA reusable workflow the yaml file **must** reside directly in the path `.github/workflows`. Therefore, subfolders cannot be used to organize files in this path.
+
+## Decision
+
+To alleviate confusion and improve on organizing GHA reusable workflows in the path `.github/workflows`, it was decided to add a prefix to the filename to provide context of the yaml file's purpose.
+
+- ***sub-***
+  - Represent a grouping of related jobs. Defines a collection of jobs, steps, and call on other reusable workflow.
+- ***reusable-***
+  - Represent an atomic piece of work executed in a Continuous Integration Continous Delivery (CICD) pipeline. It can be a collection of jobs and steps.
```

Comment:
Would it also be accurate to say "Represents an atomic piece of work... which is utilized more than once."?

Reply:
Oh, also elsewhere we've used "CI/CD", and one of the instances of "Continuous" is misspelled.